tox.ini

```diff
@@ -10,7 +10,8 @@ deps =
     mock
     pytest
     pytest-cov
-commands =
+    flake8
+commands = flake8
```

Comment:
Hm, I can't reproduce it anymore so it was probably supertitious of me. Lemme do the change. :)